engine/src/main/java/org/terasology/engine/LoggingContext.java

```diff
@@ -74,7 +77,7 @@ public static void initialize(Path logFileFolder) {
         System.setProperty(LOG_FILE_FOLDER, pathString);
 
         try {
-            deleteLogFiles(logFileFolder, 5 * 24 * 60 * 60); // JAVA8: Duration.ofDays(5).getSeconds();
+            deleteLogFiles(logFileFolder, ((int) Duration.ofDays(5).getSeconds()));
```

Comment:
Maybe `deleteLogFiles` could take a `long` parameter? This could avoid the need to downcast without reason.